cmd/containerd-shim-darwin-v1/main.go

```diff
@@ -0,0 +1,31 @@
+// +build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/containerd/containerd/runtime/v2/shim"
+)
+
+const (
+	RunuRoot = "/var/run/containerd/runu"
+)
+
+func main() {
+	shim.Run("io.containerd.darwin.v1", New)
```

Comment:
I'd prefer this to be called `io.containerd.runu.v1` and maintained in runu's repo

Reply:
Or just extend the existing `io.containerd.runc.v2` to support Darwin

Reply:
agree; will move those parts (cmd/containerd-shim-darwin-v1) to runu repo.

Reply:
did remove cmd/containerd-shim-darwin-v1.  I will update the runu part later.